label_studio/tasks/serializers.py

```diff
@@ -403,14 +401,28 @@ def add_predictions(self, task_predictions):
                         prediction_score = None
 
                 last_model_version = prediction.get('model_version', 'undefined')
-                db_predictions.append(
-                    Prediction(
-                        task=self.db_tasks[i],
-                        result=result,
-                        score=prediction_score,
-                        model_version=last_model_version
+                if flag_set(
+                    'fflag_perf_back_lsdv_4695_update_prediction_query_to_use_direct_project_relation',
```

Comment:
this code will run in rqworker and user=auto is not working there.. 
we need to extract somewhere in the beginning organization - 
`user.active_organization.created_by` and use this instead of 'auto'

Reply:
This flag is no longer needed as the flagged behavior needs to be the default and is already in develop.